src/main/java/com/epam/rd/autotasks/catalog/converter/EnumConverter.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.rd.autotasks.catalog.converter;
+
+import com.epam.rd.autotasks.catalog.constant.SortEnum;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EnumConverter implements Converter<String, SortEnum> {
+    @Override
+    public SortEnum convert(String source) {
+        try {
+            return SortEnum.valueOf(source.toUpperCase());
+        } catch (Exception E) {
+            return null;
```

Comment:
Exception E -  variables should be named in lower case
It is good practice to log exception before skipping. Please add it.